tests/gtests/graph/unit/backend/graph_compiler/core/test_conv3d.cpp

```diff
@@ -72,8 +72,8 @@ void check_conv_fwd_correctness(conv_fwd_config_t cfg,
     }
 
     sc_graph_t g;
-    auto src_dtype = sc_data_traits_t<src_type>::type();
-    auto wei_dtype = sc_data_traits_t<wei_type>::type();
+    auto src_dtype = sc_data_traits_t_t<src_type>::type();
+    auto wei_dtype = sc_data_traits_t_t<wei_type>::type();
```

Comment:
It looks like a mistake of global replacement.

Reply:
Thanks for the catch, updated.